helm-charts/splunk-otel-collector/templates/clusterRole.yaml

```diff
@@ -3,6 +3,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   name: {{ template "splunk-otel-collector.fullname" . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
This is a cluster scoped object, no need to add namespace here.

Reply:
Removed it, but is there a particular reason to not include a namespace in cluster scoped objects?

Reply:
- I've seen the namespace metadata be ignored or cause Helm manifest validation errors.
- I know this would cause Helm validation errors in our EKS add-on environment.